src/statsig/types.ts

```diff
@@ -35,6 +35,7 @@ export enum StatsigFeatureGates {
   SHOW_ASSET_DETAILS_SCREEN = 'show_asset_details_screen',
   RESTRICT_SUPERCHARGE_FOR_CLAIM_ONLY = 'restrict_supercharge_for_claim_only',
   USE_VIEM_FOR_SWAP = 'use_viem_for_swap',
+  USE_NEW_RECIPIENT_SCREEN = 'use_new_recipient_screen',
```

Comment:
How does this feature gate interact with the `USE_NEW_SEND_FLOW` gate? Are they independent? Or if the new recipient screen is shown, should the new send flow always be shown?

Reply:
Yeah, `USE_NEW_RECIPIENT_SCREEN` should imply the rest of the new send flow being used; we can set that up in Statsig, I believe.